.github/workflows/ci.yml

```
@@ -1,74 +1,100 @@
+      - name: Install dependencies
+        run: yarn install --immutable
+
       - name: Install Docs
-        run: (cd docs && yarn install --frozen-lockfile)
+        run: (cd docs && yarn install --immutable)
+
       - name: Build docs
         run: yarn run docs:build
+
       - name: Deploy docs
         uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)